WISE-Community/WISE
# Styled the project library and updated Site header and account menu links

To test:
- Log in as a teacher and select the Browse WISE Units tab on the teacher homepage. Load the three different libraries and make sure projects show up correctly search and filter work.
- Navigate around the Site and make sure header and account menu links are updated correctly depending on user role (student/teacher/researcher/admin) and router path.

Closed #1429 and #1237.
